cpp/tests/prims/mg_copy_v_transform_reduce_inout_nbr.cu

```diff
@@ -64,7 +64,8 @@ struct property_transform
     operator()(const vertex_t& val)
   {
     cuco::detail::MurmurHash3_32<vertex_t> hash_func{};
-    auto value = hash_func(val) % mod;
+    // auto value = hash_func(val) % mod;
+    auto value = 1;
```

Comment:
What's the motivation for this change? And if you intended to make this change, better delete the commented out code.